### ISSUE TITLE
Non-profit and Second Hand Shop icons

### ISSUE THREAD

**Issue by quincylvania** [openstreetmap/iD#6813]
Posted by @joni909 at https://github.com/osmlab/osm-community-index/issues/251:

> I wondered I someone could make icons for non-profit and second hands,
> because they don't really fit in with commercial nor with retail.
> 
> It's just a question and a suggestion.
> cheers
> and
> keep up the good work, all of you involved

**Comment by TAQ2** [comment]
Is this something I can work on? What does the waitfor-icon mean?

**Comment by quincylvania** [comment]
> What does the waitfor-icon label mean?

@TAQ2 It means we need to find or draw a suitable icon before we can address the issue. This particular icon sounds pretty difficult.

> Is this something I can work on?

If you know how to design icons! Check out the https://github.com/ideditor/temaki project.

**Comment by quincylvania** [comment]
Closing this in favor of #7068. This lets us consider these presets' icons in a broader context.
